deployment/terraform/iam.tf

```diff
@@ -0,0 +1,24 @@
+data "aws_iam_policy_document" "ecs_task_execution_assume_role" {
+  statement {
+    effect = "Allow"
+
+    principals {
+      type        = "Service"
+      identifiers = ["ecs-tasks.amazonaws.com"]
+    }
+
+    actions = [
+      "sts:AssumeRole",
+    ]
+  }
+}
+
+resource "aws_iam_role" "ecs_task_execution_role" {
+  name               = "ecsTaskExecutionRole${title(var.project)}${title(var.environment)}"
```

Comment:
I think that project should be removed from these resource names. I also think that it'll read easier with environment earlier in the name vs. at the end.